bouncer/commands/check_wintesses.ts

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env -S pnpm tsx
+// INSTRUCTIONS
+//
+// This command takes 2 argument.
+// 1. The chain to check or HASH if you want to pass an hash to check, it can be: ETH, BTC, DOT or HASH
+// 2. the encoded call hash if HASH was used as first argument
+// It then prints the number of validator who have correctly witnessed it, and in case this number is less than the number of validators in the authority set it will
+// print the validator IDs and vanity of the one not witnessing it.
+// In order to obtain the hash you can use PolkaJS to construct the transaction that was supposed to be witnessed. You might need some external data to do so.
+// Go to PolkaJS > Developer > Extrinsic
+// from there, once you built the extrinsic you need the encoded call hash
+
+// For example: ./commands/check_witnesses.ts ETH
+// will wait for the next chainStateUpdate extrinsic for ethereum and after some blocks (2) it will check how many validator witnessed it
+
+import { blake2AsHex } from '@polkadot/util-crypto';
+import { runWithTimeout, sleep, getChainflipApi } from '../shared/utils';
+
+function hex2bin(hex0x: string) {
```

Comment:
You can just use `parseInt(someHexString, 16).toString(2)` ? Would have to strip out the 0x first still though

Reply:
This was problematic because of the max integer size in js. Which is way lower than the value represented by the hex, not sure if there is another direct way to convert hex to binary.

Reply:
I see - should at least leave a comment that this is the reason for this function. 

Also, just noticed the file has a typo, should be `check_witnesses`